### PR TITLE
Add active state for cover, input_boolean, group and device_tracker

### PR DESCRIPTION
**Pull request by snowzach**

### CODE REVIEW

**Comment by bramkragten** [comment]
See https://github.com/home-assistant/home-assistant-polymer/pull/2398

File: src/panels/lovelace/cards/hui-entity-button-card.ts [hunk 1]
```diff
@@ -96,33 +96,33 @@ class HuiEntityButtonCard extends hassLocalizeLitMixin(LitElement)
         .longPress="${longPress()}"
       >
         ${
-          !stateObj
-            ? html`
+      !stateObj
```
[the remark this answers is not in the document]
Comment (snowzach) [reply src/panels/lovelace/cards/hui-entity-button-card.ts]: I just fixed it by hand.

File: src/components/entity/state-badge.js [hunk 2]
```diff
@@ -92,7 +96,7 @@ class StateBadge extends PolymerElement {
         if (typeof brightness !== "number") {
           errorMessage = `Type error: state-badge expected number, but type of ${
             newVal.entity_id
-          }.attributes.brightness is ${typeof brightness} (${brightness})`;
+            }.attributes.brightness is ${typeof brightness} (${brightness})`;
```
[the remark this answers is not in the document]
Comment (snowzach) [reply src/components/entity/state-badge.js]: done

File: src/panels/lovelace/cards/hui-entity-button-card.ts [hunk 3]
```diff
@@ -151,7 +151,11 @@ class HuiEntityButtonCard extends hassLocalizeLitMixin(LitElement)
         ha-icon[data-domain="light"][data-state="on"],
         ha-icon[data-domain="switch"][data-state="on"],
         ha-icon[data-domain="binary_sensor"][data-state="on"],
+        ha-icon[data-domain="input_boolean"][data-state="on"],
```
[the remark this answers is not in the document]
Comment (snowzach) [reply src/panels/lovelace/cards/hui-entity-button-card.ts]: done

File: src/components/entity/state-badge.js [hunk 4]
```diff
@@ -29,7 +29,11 @@ class StateBadge extends PolymerElement {
         ha-icon[data-domain="light"][data-state="on"],
         ha-icon[data-domain="switch"][data-state="on"],
         ha-icon[data-domain="binary_sensor"][data-state="on"],
+        ha-icon[data-domain="input_boolean"][data-state="on"],
```
[the remark this answers is not in the document]
Comment (snowzach) [reply src/components/entity/state-badge.js]: done

**Comment by snowzach** [comment]
@bramkragten are you saying I should be using custom CSS? I am looking at this and the linked ticket I think are saying this should be done with themes. Themes don't seem to let you set the colors for groups (from what I've been able to see) and I have played around and it seems like injecting CSS doesn't seem to work. I would be fine if I could make these css changes myself but I'm not aware of a way to do so.

**Comment by balloob** [comment]
As per linked PRs, we're not going to merge this right now, we need to make a decision how we want to deal with state icon colors first.

**Comment by snowzach** [comment]
Is there any way to include css to style the elements via a theme or even just css? That seems to be a drastic downside to styling with the shadow dom that if it's not a variable, you can't really style it globally from what I've seen. I've tried everything that I can think of in css and a little in javascript.

**Pull request closed without merging**